llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -912,25 +970,28 @@ class VPRecipeWithIRFlags : public VPRecipeBase {
     } else if (auto *Op = dyn_cast<FPMathOperator>(&I)) {
       OpType = OperationType::FPMathOp;
       FMFs = Op->getFastMathFlags();
+    } else {
+      OpType = OperationType::Other;
+      AllFlags = 0;
```

Comment:
Is this `else` a general fix or related to this patch?

Reply:
Yes that could be done, at the cost of slightly increasing VPRecipeBase's size.